src/devices/nm-acd-manager.c

```diff
@@ -54,10 +51,14 @@ static guint signals[LAST_SIGNAL] = { 0 };
 
 typedef struct {
 	int            ifindex;
-	guint8         hwaddr[ETH_ALEN];
+	guint8         *hwaddr;
+	size_t         hwaddr_len;
```

Comment:
guint? Because at other places, the length is limited to guint type. Or, adjust the other places.